pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/go/GoInstanceConfig.java

```diff
@@ -27,6 +27,8 @@
 @Getter
 public class GoInstanceConfig {
     private String pulsarServiceURL = "";
+    private String stateStorageServiceUrl = "";
+    private String pulsarWebServiceUrl = "";
```

Comment:
In order for these to be useful, do we need to add them to the function runtime here:

https://github.com/apache/pulsar/blob/f04252b84a8b594ea371645e367c491aa8e2dd80/pulsar-function-go/pf/instanceConf.go#L31-L45

Reply:
Resolved!

Reply:
I think I am missing something, how did you resolve this?

Reply:
You’re right, I resolved the wrong comment. I’ll get this in in a couple hours.

Reply:
Okay. NOW resolved. 😓